porcupaine/porcupaine_score.py

```diff
@@ -0,0 +1,69 @@
+from statistics import median
+
+import pandas as pd
+
+from porcupaine.settings import *
+from porcupaine.textual_model.text_model import demo
+from porcupaine.numerical_model.process_eval_new_numerical_data import predict_project_success
+
+
+def compute_porcupaine_score(name: str, description: str, public_interest: str, district: str, category: str, budget: int) -> float:
+    """
+    Args:
+        parameters of the project used to compute the porcupAIne score
+
+    Returns (float): The porcupAIne score, the chance of the project to be successful.
+                     Displays in the app after form submission.
+    """
+    num_inputs = pd.DataFrame({
+        'category': category,
+        'district': district,
+        'budget': budget
+    }, index=[0])
+    model_path = MODELS_DIR / "numerical_logistic_regression_model.pkl"
+    num_score = predict_project_success(num_inputs, model_path)
+
+    text_score = demo(name, description, public_interest)[0][1]     # the function returns the np.array of probabilities for each class, we want the proba of the class 1
```

Comment:
if you push name like `demo`, just leave comment `# FIXME rename demo to proper function name`